Update autres-methodes.md

AntoineGarciaSuarez opened the pull request:
add a point to Observation (test)

AntoineGarciaSuarez committed:
Update autres-methodes.md

add a point to Observation (test)

cmaczo commented:
Thanks Antoine!

Should other headings, like "Tests multivariés" also be followed by a `.`?

AntoineGarciaSuarez committed:
Update autres-methodes.md

Deleted the dot after Observation (Observation.)
Actually, this small add was an example during my github session with Adreanne

AntoineGarciaSuarez committed:
Update autres-methodes.md

Changed - Comment commencer- to -Par où commencer?-
How to get started? was still in english. I translated.

AntoineGarciaSuarez committed:
Update autres-methodes.md

edited the link https://www.uxmatters.com/mt/archives/2014/01/participatory-observation.php.
cause it didn't work
